test/unit/api/solver_black.cpp

```diff
@@ -1423,7 +1423,7 @@ TEST_F(TestApiBlackSolver, getUnsatCore3)
   {
     d_solver.assertFormula(t);
   }
-  Result res = d_solver.checkSat();
+  CVC4::api::Result res = d_solver.checkSat();
```

Comment:
Is this change really necessary? There is a `using namespace api` at the top of the file.

Reply:
I copied what was going on in the other files. There is some ambiguity on which `Result` to use because of the namespaces.